superbench/config/azure_ndmv4.yaml

```diff
@@ -33,6 +33,8 @@ superbench:
           node_num: 1
           env:
             NCCL_ASYNC_ERROR_HANDLING: '0'
+      frameworks:
+        - pytorch
```

Comment:
why do we need this?

Reply:
It's a bug in previous config file. The dist-inference benchmark is registered as PyTorch run mode, without this it cannot run. For cpp mode it's another code path but still need this name.

Reply:
Could we fix it directly?

Reply:
Also OK like this.

Reply:
It's a hard-code prefix for PyTorch-related benchmark, not easy to fix